doc/user/content/sql/alter-role.md

```diff
@@ -51,7 +51,7 @@ Unlike PostgreSQL, Materialize does not use role attributes to determine a roles
 top level objects such as databases and other roles. Instead, Materialize uses system level
 privileges. See [GRANT PRIVILEGE](../grant-privilege) for more details.
 
-When RBAC is enabled a role must have the `CREATEROLE` system privilege to alter another role.
+A role must have the `CREATEROLE` system privilege to alter another role.
```

Comment:
Thanks for catching these! 
Just curious ... we do have a Privileges section below (although, that they are so down at the end of the page is a bit odd).  Is it because that section is at the end where it is easy to overlook the reason for repeating this?

Reply:
Added https://github.com/MaterializeInc/materialize/pull/28667

Reply:
Oh...this page was just written up by an Engineer. 😂  Good spot, I'll move it down.

Reply:
For situations like this, you can push directly to my branch, rather than open a new PR! [Here's the workflow](https://tighten.com/insights/adding-commits-to-a-pull-request/) (it'll be useful as you work on documentation written by someone else).

Reply:
Ah, thanks much.  I thought about adding a patch but didn't want to be pushy (pun not intended).